### ISSUE TITLE
image_io.cpp:339] Could not open or find file

### ISSUE THREAD

**Issue by saiki0007** [facebook/C3D#449]
I am trying to train UCF-101 dataset on the C3D-v1.1 but i face a weird error. The frame which is being accessed is one number more than the last frame as shown below:

I0729 16:32:09.606221 526955 solver.cpp:273] Solving C3D-UCF101Net
I0729 16:32:09.606225 526955 solver.cpp:274] Learning Rate Policy: step
E0729 16:32:09.634727 526978 image_io.cpp:339] Could not open or find file UCF-101/Hammering/v_Hammering_g13_c07//000020.jpg
I0729 16:32:09.642668 526955 blocking_queue.cpp:49] Waiting for data
E0729 16:32:09.661631 526978 image_io.cpp:339] Could not open or find file UCF-101/Skijet/v_Skijet_g19_c03//000044.jpg
E0729 16:32:09.716790 526978 image_io.cpp:339] Could not open or find file UCF-101/RockClimbingIndoor/v_RockClimbingIndoor_g09_c01//000052.jpg
E0729 16:32:09.747853 526978 image_io.cpp:339] Could not open or find file UCF-101/StillRings/v_StillRings_g18_c01//000044.jpg
E0729 16:32:09.804396 526978 image_io.cpp:339] Could not open or find file UCF-101/Bowling/v_Bowling_g19_c01//000025.jpg
E0729 16:32:09.822487 526978 image_io.cpp:339] Could not open or find file UCF-101/SoccerJuggling/v_SoccerJuggling_g19_c05//000052.jpg
E0729 16:32:09.889794 526978 image_io.cpp:339] Could not open or find file UCF-101/WallPushups/v_WallPushups_g19_c02//000019.jpg
E0729 16:32:09.964848 526978 image_io.cpp:339] Could not open or find file UCF-101/PlayingTabla/v_PlayingTabla_g17_c02//000054.jpg
E0729 16:32:10.011976 526978 image_io.cpp:339] Could not open or find file UCF-101/HulaHoop/v_HulaHoop_g17_c03//000022.jpg
E0729 16:32:10.023752 526978 image_io.cpp:339] Could not open or find file UCF-101/CleanAndJerk/v_CleanAndJerk_g12_c07//000032.jpg
E0729 16:32:10.075891 526978 image_io.cpp:339] Could not open or find file UCF-101/RockClimbingIndoor/v_RockClimbingIndoor_g08_c05//000098.jpg
E0729 16:32:10.137146 526978 image_io.cpp:339] Could not open or find file UCF-101/CuttingInKitchen/v_CuttingInKitchen_g13_c01//000031.jpg
E0729 16:32:10.144898 526978 image_io.cpp:339] Could not open or find file UCF-101/WritingOnBoard/v_WritingOnBoard_g20_c07//000024.jpg
E0729 16:32:10.206794 526978 image_io.cpp:339] Could not open or find file UCF-101/HandstandWalking/v_HandstandWalking_g09_c02//000031.jpg
E0729 16:32:10.244976 526978 image_io.cpp:339] Could not open or find file UCF-101/ApplyEyeMakeup/v_ApplyEyeMakeup_g12_c03//000030.jpg
E0729 16:32:10.256954 526978 image_io.cpp:339] Could not open or find file UCF-101/Haircut/v_Haircut_g09_c01//000046.jpg
E0729 16:32:10.395485 526978 image_io.cpp:339] Could not open or find file UCF-101/PlayingSitar/v_PlayingSitar_g17_c07//000094.jpg
E0729 16:32:10.517381 526978 image_io.cpp:339] Could not open or find file UCF-101/RockClimbingIndoor/v_RockClimbingIndoor_g17_c02//000083.jpg
I0729 16:33:49.253451 526955 solver.cpp:219] Iteration 0 (0 iter/s, 99.647s/20 iters), loss = 4.53648
I0729 16:33:49.253582 526955 solver.cpp:238]     Train net output #0: loss = 4.53648 (* 1 = 4.53648 loss)
I0729 16:33:49.253629 526955 sgd_solver.cpp:105] Iteration 0, lr = 0.01
E0729 16:33:49.530026 526978 image_io.cpp:339] Could not open or find file UCF-101/Archery/v_Archery_g13_c07//000040.jpg
E0729 16:33:49.591573 526978 image_io.cpp:339] Could not open or find file UCF-101/PlayingFlute/v_PlayingFlute_g05_c01//000066.jpg
E0729 16:33:49.605487 526978 image_io.cpp:339] Could not open or find file UCF-101/ParallelBars/v_ParallelBars_g15_c04//000033.jpg
E0729 16:33:49.616542 526978 image_io.cpp:339] Could not open or find file UCF-101/PlayingTabla/v_PlayingTabla_g13_c02//000060.jpg
E0729 16:33:49.678442 526978 image_io.cpp:339] Could not open or find file UCF-101/PlayingDaf/v_PlayingDaf_g08_c03//000063.jpg
E0729 16:33:49.690623 526978 image_io.cpp:339] Could not open or find file UCF-101/PommelHorse/v_PommelHorse_g09_c01//000043.jpg

For example, in the path UCF-101/PommelHorse/v_PommelHorse_g09_c01/ the total frames that are present are 42 but frame number 43 is being opened. This is similar to all the errors.

Please tell me how to solve this issue as I am new to Caffe.

**Comment by saiki0007** [comment]
I have also checked the previously reported similar issues but couldn't solve it.
